### PR TITLE
BXMSDOC-4515: Added recommendation for PerProcessInstance in managing and monitoring process server doc

### PR DESCRIPTION
**Pull request by hmanwani-rh**
See JIRA: https://issues.redhat.com/browse/BXMSDOC-4515

Doc previews:
[Managing and monitoring KIE Server RHPAM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-4515-RHPAM/#deployment_descriptor_configuration)
[Managing and monitoring KIE Server RHDM](http://file.pnq.redhat.com/~hmanwani/BXMSDOC-4515-RHDM/#deployment_descriptor_configuration)

### CODE REVIEW

**Review by cristianonicolai — APPROVED** [review]

**Review by kaldesai — APPROVED** [review]
Hey @hmanwani-rh, Looks good to me.

File: doc-content/enterprise-only/admin-and-config/deployment-descriptor-configuration-proc.adoc [hunk 1]
```diff
@@ -104,4 +104,4 @@ The following table lists these along with the permissible and default values fo
 
 |===
 
-WARNING: Do not use the Singleton runtime strategy with the EJB Timer Scheduler (the default scheduler in {KIE_SERVER}) in a production environment. This combination can result in Hibernate problems under load. For more information about this limitation, see https://access.redhat.com/solutions/4065521[Hibernate issues with Singleton strategy and EJBTimerScheduler].
+WARNING: Do not use the Singleton runtime strategy with the EJB Timer Scheduler (the default scheduler in {KIE_SERVER}) in a production environment. This combination can result in Hibernate problems under load. `PER_PROCESS_INSTANCE` is recommended for runtime strategy if there is no specific reason to use other strategies. For more information about this limitation, see https://access.redhat.com/solutions/4065521[Hibernate issues with Singleton strategy and EJBTimerScheduler].
```
Comment (MarianMacik) [inline doc-content/enterprise-only/admin-and-config/deployment-descriptor-configuration-proc.adoc]: I would just rephrase it a little bit to `` Per process instance runtime strategy is recommended if...``

Comment (hmanwani-rh) [reply doc-content/enterprise-only/admin-and-config/deployment-descriptor-configuration-proc.adoc]: @MarianMacik, thanks!
I have updated the sentence. Please review

**Review by MarianMacik — APPROVED** [review]
Looks good to me now. Thanks!

**Pull request merged**